### DISCUSSION TITLE
Code for beginner

### DISCUSSION THREAD

**Discussion by bvrilliance** [Hammerspoon/hammerspoon#2769] [Q&A]
Hi, can you help me to code so that when I press 'down', it results in pressing 'down'+'left'?

**Comment by latenitefilms** [comment]
If you want down and left to be pressed at the same time, something like this should work:

```lua
hs.hotkey.new({}, "down", function()
  print("down pressed")
  
  hs.eventtap.event.newKeyEvent("down", true):post()
  hs.eventtap.event.newKeyEvent("left", true):post()
  
  hs.eventtap.event.newKeyEvent("down", false):post()
  hs.eventtap.event.newKeyEvent("left", false):post()
end)
```

If you want it to press down THEN left, you could do something like:

```lua
hs.hotkey.bind({}, "down", function()
  print("down pressed")
  
  hs.eventtap.keyStroke({}, "down", 0)
  hs.eventtap.keyStroke({}, "left", 0)  
end)
```

**Reply by bvrilliance** [reply]
Hi, thank you!
I tried them both, the first one doesn't work for me, while the second one is just executing 'left'. In a way 'down'='left'.

**Reply by latenitefilms** [reply]
Maybe have a look at https://karabiner-elements.pqrs.org - as it works at a much deeper level than Hammerspoon.
